### DISCUSSION TITLE
May I know when approximately the next release ?

### DISCUSSION THREAD

**Discussion by pavel-mitchenko** [ravibpatel/AutoUpdater.NET#665] [Q&A]
May I know when  approximately  the NuGet package with  [fix ](https://github.com/ravibpatel/AutoUpdater.NET/commit/8e878844719bc0aa750b6ab20cd39b00c5ada059) will be released ?

**Comment by ravibpatel** [comment]
I pushed v1.8.5 to NuGet. It should be available soon.

**Reply by pavel-mitchenko** [reply]
Than you very much ! I will let you know in case of other issues with it
